hustle/jobs/models.py

```diff
@@ -13,7 +13,7 @@ def __str__(self):
 
 
 class Job(models.Model):
-    time_estimate = models.IntegerField(name="time_estimate")
+    time_estimate = models.IntegerField("Time estimate (in hours)", name="time_estimate")
```

Comment:
If in hours, perhaps we ought to change it from an `IntegerField` to a `DecimalField` to allow half or quarter hours.